SU2_CFD/src/solver_direct_mean_inc.cpp

```diff
@@ -7323,6 +7545,42 @@ CIncNSSolver::CIncNSSolver(CGeometry *geometry, CConfig *config, unsigned short
     }
   }
 
+  VertexTraction = NULL;
```

Comment:
Yep, just moved the initialization to CSolver in d3bc9b5ba. Instead of adding a function to the inline, given that we are moving already the inlines to the declarations as in #725, I declared the function directly in solver_structure.hpp, which soon should become CSolver.hpp